src/Makefile.am

```diff
@@ -106,6 +106,7 @@ BITCOIN_CORE_H = \
   miner.h \
   net.h \
   netbase.h \
+  netaddress.h \
```

Comment:
nit: Please move `netaddress.h` one line up to match the alphabet and the `cpp` files order below.